crates/api/src/lib.rs

```diff
@@ -1539,9 +1508,10 @@ mod test {
     struct TestDispatch<'a> {
         api: ApiDispatch,
         _db: TemporaryDatabase<'a>, // share lifetime
+        _tp: EmbeddedChronicleTp,
     }
 
-    impl TestDispatch<'_> {
+    impl<'a> TestDispatch<'a> {
```

Comment:
Feels like a no-op but no objection.

Reply:
It is, I'll clippy it out after this is in main